WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -249,7 +245,7 @@ - (RemoteBlogSettings *)remoteBlogSettingFromJSONDictionary:(NSDictionary *)json
     //
     if ([[rawSettings numberForKey:BlogRemoteDefaultPostFormatKey] isEqualToNumber:@(0)] ||
         [[rawSettings stringForKey:BlogRemoteDefaultPostFormatKey] isEqualToString:@"0"]) {
-        remoteSettings.defaultPostFormat = PostFormatStandard;
+        remoteSettings.defaultPostFormat = @"standard";
```

Comment:
@koke I was trying to remove any dependency of the Model objects from the Remote classes. The constant is being defined and also used on the Model object. 
I can redefine it here for the remote class do you think it's better?